types/presto.d.ts

```diff
@@ -1,205 +1,7334 @@
 /**
- * Taken from @see {@link https://demo.castlabs.com/#/docs?q=clpp.Player}
+ * These typings will shortly become available on NPM as part of `@castlabs/prestoplay`.
```

Comment:
:rocket:

Reply:
Will upgrade to the latest version of PRESTO and delete this file altogether as Thasso pointed out.